sdk/src/wallet/error.rs

```diff
@@ -37,6 +37,8 @@ pub enum Error {
     /// Custom input error
     #[error("custom input error {0}")]
     CustomInput(String),
+    #[error("delay delegation claiming failed {0}")]
+    DelegationTransitionFailed(String),
```

Comment:
Also, you only use it for delegation output not found, maybe make it a bit more specific to it

Reply:
true